internal/ui/login/handler/mfa_init_u2f.go

```diff
@@ -0,0 +1,57 @@
+package handler
+
+import (
+	"encoding/base64"
+	"net/http"
+
+	"github.com/caos/zitadel/internal/auth_request/model"
+)
+
+const (
+	tmplMfaU2FInit = "mfainitu2f"
+)
+
+func (l *Login) renderRegisterU2F(w http.ResponseWriter, r *http.Request, authReq *model.AuthRequest, err error) {
+	var errType, errMessage string
+	if err != nil {
+		errMessage = l.getErrorMessage(r, err)
+	}
+	u2f, err := l.authRepo.AddMfaU2F(setContext(r.Context(), authReq.UserOrgID), authReq.UserID)
+	if err != nil {
+		l.renderError(w, r, authReq, err)
```

Comment:
i think this should render register u2 again with error message?
Maybe also in the following parts where we habe renderError